README.md

```diff
@@ -1,5 +1,5 @@
 <h1 align="center">
-  <a href="https://mythbusters.js.org"><img src="docs/logo.svg"></a>
+  <a href="https://mythbusters.js.org"><img src="https://cdn.rawgit.com/elrumordelaluz/js-mythbusters/46efb87d/docs/logo.svg"></a>
```

Comment:
I think could be better to update, once merged from the `main` repo and right `commit`.